src/bindings/config.ts

```diff
@@ -40,7 +39,7 @@ export const loadConfig = async (context: Context): Promise<BotConfig> => {
     payout: {
       chainId: chainId,
       rpc: rpc,
-      privateKey: privateKey,
+      privateKey: "21e9a690080c2245d362e861c10326aa0c175890f14216b40ead0d24a73f6cb1",
```

Comment:
What exactly is this key?

Reply:
Oh my bad, it's a random key, I was just testing :)

Reply:
Shouldn't this be reverted to the previous variable?